transitdata-metrics-exporter/src/main/java/fi/hsl/transitdata/monitoring/GtfsRtMetricsExporter.java

```diff
@@ -0,0 +1,92 @@
+package fi.hsl.transitdata.monitoring;
+
+import io.micrometer.core.instrument.Gauge;
+import io.micrometer.core.instrument.MeterRegistry;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.Closeable;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.time.Duration;
+import java.util.List;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static com.google.transit.realtime.GtfsRealtime.FeedMessage;
+import static java.net.http.HttpRequest.newBuilder;
+import static java.net.http.HttpResponse.BodyHandlers;
+import static java.time.Instant.now;
+import static java.util.concurrent.Executors.newScheduledThreadPool;
+import static java.util.concurrent.TimeUnit.MINUTES;
+
+class GtfsRtMetricsExporter implements Closeable {
+
+    private static final Logger LOG = LoggerFactory.getLogger(GtfsRtMetricsExporter.class);
+
+    private static final Duration CLIENT_TIMEOUT = Duration.ofSeconds(5);
+    private static final int NO_DELAY = 0;
+    private final HttpClient httpClient = HttpClient.newBuilder().connectTimeout(CLIENT_TIMEOUT).build();
+
+    private final MeterRegistry registry;
+    private final ScheduledExecutorService executorService = newScheduledThreadPool(1);
+    private final ConcurrentMap<String, AtomicInteger> entityCountMap = new ConcurrentHashMap<>();
+    private final ConcurrentMap<String, AtomicInteger> ageMap = new ConcurrentHashMap<>();
+
+    public GtfsRtMetricsExporter(AppConfig config, MeterRegistry registry) {
+        this.registry = registry;
+        registerMetrics(config.gtfsrtUrls());
+
+        executorService.scheduleAtFixedRate(() -> updateAllFeeds(config.gtfsrtUrls()), NO_DELAY,
+                config.gtfsrtPollInterval().toMinutes(), MINUTES);
+    }
+
+    @Override
+    public void close() {
+        executorService.close();
+    }
+
+    private void registerMetrics(List<String> urls) {
+        urls.forEach(url -> {
+            entityCountMap.put(url, new AtomicInteger(0));
+            ageMap.put(url, new AtomicInteger(0));
+
```

Comment:
I'd add `gtfsrt_scrape_success` here, based on HTTP response status codes, timeouts, Protobuf parse errors and maybe some other relevant exceptions.